tools/src/tensil/tools/compiler/MemoryManager.scala

```diff
@@ -7,38 +7,34 @@ import java.io._
 import scala.collection.mutable
 import org.tensorflow.framework.types.DataType
 
-import _root_.tensil.tools.data.{Shape, TensorData}
-import _root_.tensil.tools.util
-import _root_.tensil.tools.{
-  CompilerException,
-  TraceContext,
-  TracepointCondition
+import tensil.tools.data.{Shape, TensorData}
+import tensil.tools.util
+import tensil.tools.{CompilerException, TraceContext, TracepointCondition}
+import tensil.{Architecture, ArchitectureDataType}
+
+object MemoryManager {
+  object ReservedConsumers {
+    val Output = "~output~"
+    val Consts = "~consts~"
+
+    val All = Seq(Output, Consts)
+  }
 }
-import _root_.tensil.{Architecture, ArchitectureDataType}
 
 class MemoryManager(
+    tempSpace: MemorySpace,
```

Comment:
can you remind me what we put in `temp` again?

Reply:
We use temp memory in frontends for memory that during scheduling will be mapped to accumulators. Usually temp is intermediate results between convolution/dense and activation function.